src/components/Item.js

```diff
@@ -1,11 +1,14 @@
 import React, { useEffect, useState } from 'react';
 import { updateDoc, serverTimestamp, doc } from 'firebase/firestore';
 import { db } from '../lib/firebase';
+import { calculateEstimate } from '@the-collab-lab/shopping-list-utils';
 import './Item.css';
 
 function Item({ item, userToken }) {
   const [checked, setChecked] = useState(false);
-  const [daysSincePurchased, setDaysSincePurchased] = useState();
+  const [daysSincePurchased, setDaysSincePurchased] = useState(0);
+  const [itemBackup, setItemBackup] = useState(item);
```

Comment:
Since you don't ever reset this value you won't need the setter
```suggestion
  const [itemBackup] = useState(item);
```

Reply:
That said, as @JordanRaleigh pointed out in his comment this gives you a very limited backup in that if the user leaves and/or refreshes the page that value will change to the updated value.

I'll leave further thoughts about this on his comment to keep the conversation together.